src/Tests/UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -2601,5 +2602,54 @@ private void TestSupportedOSPlatformWarnings()
             WarningStatic.WarningEvent += (object s, Int32 v) => { }; // warning CA1416
         }
 #endif
+
+        [Fact]
+        private void TestPnpProperties()
+        {
+            TestPnpPropertiesAsync().Wait(10000);
+        }
```

Comment:
Make the test `async` instead of doing sync-over-async.

```suggestion
        private async Task TestPnpProperties()
        {
            await TestPnpPropertiesAsync();
        }
```

Reply:
tests can't be async - the runner will terminate before they finish.  we have to use this pattern.

Reply:
xUnit supports `async Task` tests and keeps the runner alive until the tasks terminate. We use this throughout dotnet/runtime.